account_invoice_shipping_address/__openerp__.py

```diff
@@ -23,9 +23,9 @@
 
 {
     "name": "Invoice Shipping Address",
-    'summary': """
+    'description': """
```

Comment:
From Odoo v8, the description is automatically binded from the README.rst file, so no need to duplicate it.